addons/web/static/src/core/macro.js

```diff
@@ -125,6 +126,7 @@ export class Macro {
     }
 
     async start() {
+        this.listenBeforeUnload();
```

Comment:
I would comment this to be able to merge this PR smoothly, and next week, try to uncomment it (when there'll be less activity on the mergebot, or with p0, after the freeze)